### ISSUE TITLE
React style guide

### ISSUE THREAD

**Issue by jarandmi** [agensdev/playbook-web#2]
Maybe we should move https://github.com/jarandmi/React-style-guide to this playbook?

**Comment by Haraldson** [comment]
I’m all for it. Perhaps we should also discuss JavaScript code style more in general.

**Comment by jarandmi** [comment]
Yes, agree. We should make a style guide for javascript, and another guide for the react/JSX specific code, like Airbnb does it:

https://github.com/airbnb/javascript
https://github.com/airbnb/javascript/tree/master/react
